data/campaigns/World_Conquest/gui/help_dialog.cfg

```diff
@@ -178,7 +178,7 @@
                                                                                 [label]
                                                                                     definition = "default_large"
                                                                                     id = "training_caption"
-                                                                                    label = "default description"
+                                                                                    label = _ "default description"
```

Comment:
I would replace all of these with empty strings instead, or maybe even remove these lines entirely. There's no point throwing a "default description" string into the po files that will never be seen in the game.

Reply:
Replaced it with empty strings instead.